grass-gis-addons/m.analyse.buildings/r.extract.buildings/r.extract.buildings.py

```diff
@@ -71,36 +66,37 @@
 # %option
 # % key: max_fd
 # % type: double
-# % required: no
+# % required: yes
 # % multiple: no
 # % label: Maximum value of fractal dimension of identified objects (see v.to.db)
 # % answer: 2.1
 # % guisection: Parameters
 # %end
 
 # %option
-# % key: ndvi_thresh
+# % key: ndvi_perc
 # % type: integer
 # % required: no
 # % multiple: no
-# % label: Define fix NDVI threshold (on a scale from 0-255) instead of estimating it from FNK
+# % label: NDVI percentile in vegetated areas to use for thresholding
+# TODO: make radio button and add default
+# # % answer: 5
 # % guisection: Parameters
 # %end
 
 # %option
-# % key: ndvi_perc
+# % key: ndvi_thresh
 # % type: integer
 # % required: no
 # % multiple: no
-# % label: NDVI percentile in vegetated areas to use for thresholding
+# % label: Fix NDVI threshold (on a scale from 0-255) instead of estimated value from ndvi_perc and FNK
+# TODO: make radio button and add default
+# # % answer: 145
 # % guisection: Parameters
 # %end
 
 # %option G_OPT_V_OUTPUT
-# % key: output
-# % type: string
-# % required: yes
-# % label: Name for output vector map
+# % answer: buildings_Bottrop_2017
```

Comment:
shouldn't it be `buildings` or somehting like this. Not region specific

Reply:
Good question. It was wished for default values according to the documentation. However the example is from `r.trees.peak` and `r.tree.mlapply`, there the default values are more generic like `tree_peaks`.

We could either do the defaults like in the documentation (also region specific) or adjust the examples in the documentation as well and use non-region-specific defaults.

Reply:
Yes, I agree. I changed the defaults to something more generic. A specific example can be added to the documentation.